ROM.sh

```diff
@@ -1440,7 +1440,7 @@ function the_start() # 0
 
     if [[ "$BRANCH" =~ (master|staging) ]]; then
         # Download the Remote Version of Updater, determine the Internet Connectivity by working of this command
-        curl -fs -o upScriBt.sh https://raw.githubusercontent.com/a7r3/ScriBt/${BRANCH}/upScriBt.sh && \
+        curl -fs -o ${PATHDIR}upScriBt.sh https://raw.githubusercontent.com/a7r3/ScriBt/${BRANCH}/upScriBt.sh && \
```

Comment:
Cool. And better that [Line 1436](https://github.com/a7r3/ScriBt/blob/master/ROM.sh#L1436] is present in [upScriBt](https://github.com/a7r3/ScriBt/blob/master/upScriBt.sh#L6) itself, I'm fine with this